Classes/YLTableViewHeaderFooterView.h

```diff
@@ -24,12 +25,15 @@
  */
 @interface YLTableViewHeaderFooterView : UIView
 
+- (nullable instancetype)initWithFrame:(CGRect)frame NS_UNAVAILABLE;
```

Comment:
@bmelts What do you think of the `nullable` return type here and in `-initWithView`?